src/Models/Model.php

```diff
@@ -96,13 +96,23 @@ protected function parse($attribute, $value)
                     break;
 
                 case 'prices':
+                    // CardMarket API Data
+                    if (property_exists($value, 'averageSellPrice')) {
+                        $class = '\\Pokemon\\Models\\CardMarketPrices';
+                        break;
+                    }
+                    // TCGPlayer API Data
```

Comment:
This can be remove since this is being overridden in the calling model, `CardMarket` in this case.
```suggestion
```